portal/src/main/scala/cromwell/pipeline/datastorage/dto/Project.scala

```diff
@@ -13,3 +14,15 @@ final case class Project(
 final case class ProjectId(value: String) extends MappedTo[String]
 
 final case class ProjectAdditionRequest(ownerId: UserId, name: String, repository: String)
+
+object Project {
+  implicit val projectFormat: OFormat[Project] = Json.format[Project]
+}
+
+object ProjectId {
+  implicit val projectIdFormat: OFormat[ProjectId] = Json.format[ProjectId]
```

Comment:
`ProjectId` is just a wrapper for a `String`. It is better to provide a custom formatter similar to how this is done for a `UserId`.

Reply:
fixed